core-plugins/src/main/java/co/cask/hydrator/plugin/batch/action/OracleExportAction.java

```diff
@@ -139,19 +136,17 @@ private String buildOracleExportCommand() {
   }
 
   private String getColSeparator(String format) {
-    String columnSeparator = new String();
     switch (SeparatorFormat.valueOf(format)) {
-      case csv:
-        columnSeparator = ",";
-        break;
-      case tsv:
-        columnSeparator = "  ";
-        break;
-      case psv:
-        columnSeparator = "|";
-        break;
+      case CSV:
+        return ",";
+      case TSV:
+        return "  ";
+      case PSV:
+        return "|";
+      default:
```

Comment:
how will it get escaped? Also, if ``format`` is other than "CSV", "TSV" or "PSV", then line 140 will throw an error, so control will never reach here